src/renderer/components/+workspaces/workspace-menu.tsx

```diff
@@ -1,66 +0,0 @@
-import "./workspace-menu.scss";
```

Comment:
These chages to workspaces seems to nulify the changes to #1927 is that true?

Reply:
Kind of, as the to-be-fixed issue states this PR refactors workspace management to command palette. We need to think again if we really need also a more traditional UI for this.